include/arch/xtensa/sys_io.h

```diff
@@ -115,6 +115,10 @@ static ALWAYS_INLINE
 	return ret;
 }
 
+static ALWAYS_INLINE void sys_write16(u16_t data, mem_addr_t addr)
```

Comment:
Make it in a separate patch.

Also, move it between sys_write/read32 and sys_write/read8, and add sys_read16 as well.

Reply:
ok